README.md

```diff
@@ -24,10 +24,13 @@ Turns will proceed clockwise from that player.
 
 ## The Game
 ### Overview
-All players begin the game as Lads. 
-A Lad ascends to the rank of Lord when his or her starting nail has been hammered completely into the stump.
-As a Lad you are only able to strike your starting nail. As a Lord you are allowed to strike the 3 center nails.
-The game ends when the 3 center nails have been hammered completely into the stump. The winner is the person who dealt the final strike to the last nail.
+All players begin the game as Lads. The max number of possible Lords is equal to the number of center nails. This is referred to as the House of Lords. When a player ascends to the rank of Lord they effectively occupy a seat in thes House of Lords. If all seats in the House of Lords are taken then Lads are unable to ascend to the rank of Lord until a Lord is demoted or an Uprising takes place.
```

Comment:
`occupy a seat in thes House of Lords` should be `occupy a seat in the House of Lords`

Reply:
`House of Lords` 👍  I support giving every aspect of this game a special name.